airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -804,7 +804,18 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         :param logger:  logger object
         :return Tuple[bool, any]: (True, None) if the input config can be used to connect to the API successfully, (False, error) otherwise.
         """
-        auth = TokenAuthenticatorBase64(token=config["api_secret"])
+        auth = None
+        try:
+            auth = TokenAuthenticatorBase64(token=config["api_secret"])
+        except KeyError as key_error:
```

Comment:
You don't have to declare the `key_error` if you're not using it. You can just do `except KeyError:`.

Reply:
removed